dynamic-layers/intel-ese-bsp/recipes-kernel/linux/linux-intel-ese-lts-acrn-sos_5.4.bb

```diff
@@ -7,6 +7,7 @@ LINUX_VERSION_EXTENSION = "-linux-intel-ese-lts-acrn-sos"
 SUMMARY = "Linux Intel ESE Kernel with ACRN enabled (SOS)"
 
 KERNEL_FEATURES_append = " features/criu/criu-enable.scc \
+                           features/docker/docker.cfg \
```

Comment:
Can you use features/docker/docker.scc instead ?

Reply:
Hi naveen,
Done, please help check, thanks